wpeverest/everest-forms
# Form template plan name

### All Submissions:






### Changes proposed in this Pull Request:
 This PR removes the PRO label and shows the plan name on the Form Templates.

### How to test the changes in this Pull Request:

1. Please go to the Add new form.
2. Previously you were seeing the PRO label on the form templates which are premium but now you can see the plan name.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Plan name shown on form templates
